packages/platform-ios/src/commands/runIOS/index.ts

```diff
@@ -128,9 +128,37 @@ async function runOnSimulator(
     throw new CLIError('Could not parse the simulator list output', error);
   }
 
-  const selectedSimulator = findMatchingSimulator(simulators, args.simulator);
-  if (!selectedSimulator) {
-    throw new CLIError(`Could not find "${args.simulator}" simulator`);
+  let simulator: string = args.simulator;
+  let selectedSimulator: {
+    udid: string;
+    name: string;
+    booted: boolean;
+    version: string;
+  } | null = null;
+
+  /**
+   * If no simulator was chosen, try simulators in following order
+   * - iPhone 11
+   * - iPhone X
+   * - iPhone 8
+   */
+  const defaultSimulators = ['iPhone 11', 'iPhone X', 'iPhone 8'];
+  if (!simulator) {
+    defaultSimulators.some(target => {
+      selectedSimulator = findMatchingSimulator(simulators, target);
+      return selectedSimulator;
+    });
+
+    if (!selectedSimulator) {
+      throw new CLIError(
+        'Could not find any of the following simulators: "iPhone 11" "iPhone X "iPhone 8"',
+      );
+    }
+  } else {
+    selectedSimulator = findMatchingSimulator(simulators, simulator);
+    if (!selectedSimulator) {
```

Comment:
I believe we can move this and another `if` statement at line `152` outside of this block. If we would place it below line 163, we would be able to keep only one `if` statement, instead of the two.